src/BlazorUI/Bit.BlazorUI/Components/Navs/Pivot/BitPivot.scss

```
@@ -271,39 +331,71 @@
+.bit-pvt-ovh {
+    display: none !important;
+}
+
+.bit-pvti-dbt {
+    border: 0;
+    padding: 0;
+    flex-shrink: 0;
+    cursor: pointer;
+    display: inline-flex;
+    align-items: center;
+    color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)